gradle/updateReleasedVersions.gradle

```diff
@@ -16,9 +16,28 @@
 
 import org.gradle.build.UpdateReleasedVersions
 import org.gradle.build.ReleasedVersion
+import groovy.json.JsonSlurper
+
+ext {
+    releasedVersionsJsonFile = file('released-versions.json')
+}
+
+tasks.withType(UpdateReleasedVersions) {
+    releasedVersionsFile = releasedVersionsJsonFile
+}
 
 task updateReleasedVersions(type: UpdateReleasedVersions) {
-    releasedVersionsFile = file('released-versions.json')
     def currentReleasedVersionProperty = project.findProperty('currentReleasedVersion')
     currentReleasedVersion = currentReleasedVersionProperty ? new ReleasedVersion(currentReleasedVersionProperty, project.findProperty('currentReleasedVersionBuildTimestamp')) : null
 }
+
+task updateReleasedVersionsToLatestNightly(type: UpdateReleasedVersions) {
+    def latestNightlyVersion = latestNightly()
```

Comment:
We should get the version lazily I think. If not, every time you run a build you will try to contact services.gradle.org. We should not do this.

Reply:
done